src/main/java/ch/kalunight/zoe/util/FullTierUtil.java

```diff
@@ -0,0 +1,91 @@
+package ch.kalunight.zoe.util;
+
+import ch.kalunight.zoe.exception.NoValueRankException;
+import ch.kalunight.zoe.model.player_data.FullTier;
+import ch.kalunight.zoe.translation.LanguageManager;
+import net.rithms.riot.api.endpoints.league.dto.LeagueEntry;
+
+public class FullTierUtil {
+
+  private FullTierUtil() {
+    // hide default public constructor
+  }
+  
+  public static String getTierRankTextDifference(LeagueEntry oldEntry, LeagueEntry newEntry, String lang) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getTierRankTextDifference(LeagueEntry, LeagueEntry, String)' has an NPath complexity of 216](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5406891)